pandasai/core/matplotlib_chart_generator.py

```diff
@@ -0,0 +1,18 @@
+from chart_generator import ChartGenerator
+
+class MatplotlibChartGenerator(ChartGenerator):
+    """
+    Matplotlib based Chart Generator
+    """
+ 
+    def generate_bar(self, df, x, y, **kwargs):
+        if "stacked" in kwargs and kwargs["stacked"] == True:
+            return df.plot.bar(**kwargs)   
+        else:
+            return df.plot.bar(x, y, **kwargs)   
+    
+    def generate_line(self, df, x, y, **kwargs):
+        return df.plot.line(x, y, **kwargs)
+
+    def generate_scatter(self, df, x, y, **kwargs):
+        return df.plot.scatter(x, y, **kwargs)
```

Comment:
Pls add an empty line at the end of a module. See [[1](https://unix.stackexchange.com/a/18789)], [[2](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)].